### PR TITLE
Host type check

### PR DESCRIPTION
**Pull request by doganoo**
This PR should introduce a new option `host_name` to duck-pony.

Given an infrastructure you are hosting a master and X slave machines. Duck-pony should not drop databases on slaves as it would result in serious issues with the database replication.

The proposed option is simple: When provided, it will check the option agains PHP's `gethostname()` method. If they are the same, the command runs as usual. If not, the command stops with a notice.

Affecting commands:

* PurgeDatabaseCommand
* PurgeIssueDatabaseCommand

### CODE REVIEW

File: src/Console/Command/Option/HostNameOptionTrait.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * @uses \Zend\Config\Config
+ * @author Dogan Ucar <dogan@dogan-ucar.de>
+ */
+trait HostNameOptionTrait
+{
+
+    protected function configureHostNameOption(): void
+    {
+        $this->addOption('host_name', 'hn', InputOption::VALUE_OPTIONAL, 'The hostname of the master host machine');
```
Comment (icanhazstring) [inline src/Console/Command/Option/HostNameOptionTrait.php]: I think given this option the `InputOption` value needs to be required.
As it doesn't make sense to leave it empty.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.6.5] - 2020-07-12
+### Changed:
```
Comment (icanhazstring) [inline CHANGELOG.md]: Can you split this into `### Added` and `### Changed`?

**Review by icanhazstring — CHANGES_REQUESTED** [review]
Nice addition. Thank you!
Some small adjustments and we are ready to merge 👍

File: src/Console/Command/Option/HostNameOptionTrait.php (hunk 1 above)
Comment (icanhazstring) [reply src/Console/Command/Option/HostNameOptionTrait.php]: Also. I don't think the short handle with two characters works. Did you check it?

Comment (doganoo) [reply src/Console/Command/Option/HostNameOptionTrait.php]: Thank you for your feedback. I changed the option's value to required. I missunderstood the `optional`, sorry!

Indead, the two charactered short handle does not work. Changed it to `h`, if this is ok for you.

**Review by icanhazstring — APPROVED** [review]
Thanks 👍

**Pull request merged**